statvar_imports/ireland_census/run.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+SCRIPT_PATH=$(realpath "$(dirname "$0")")
+
+python3 $SCRIPT_PATH/../../tools/statvar_importer/stat_var_processor.py --input_data=$SCRIPT_PATH/input_files/births.csv --pv_map=$SCRIPT_PATH/irl_birth_pvmap.csv --config_file=$SCRIPT_PATH/irl_metadata.csv --existing_statvar_mcf=gs://unresolved_mcf/scripts/statvar/stat_vars.mcf --output_path=$SCRIPT_PATH/output_files/irl_birth_output
```

Comment:
How are failures handled here? The script doesn't seem to exit with a non-zero status code when a Python command fails.